src-docs/src/views/guidelines/sass.js

```diff
@@ -113,7 +112,7 @@ function renderPaletteColor(palette, color) {
       <OuiFlexItem grow={false}>
         <div
           className="guideSass__swatch"
-          style={{ background: rgbToHex(palette[color].rgba).toUpperCase() }}
+          style={{ background: palette[color] }}
```

Comment:
@Willie-The-Lord Is `palette[color]` already uppercased, or should we keep `.toUpperCase()` for consistency?

Reply:
It's already uppercased.

Reply:
The hex color code originates from this location and is already in uppercase: https://github.com/Willie-The-Lord/oui/blob/issue1109/src/global_styling/variables/_colors.scss#L16 Should we still include `.toUpperCase()` as a precaution, in case the file changes in the future and potentially breaks something?

Reply:
nah, that's fine - thanks for the explanation